From e9e03d8a8f819a930acd3765618ba810a057e68c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 20 May 2020 20:25:06 +0200
Subject: [PATCH] generic/config: enable CONFIG_ARM_MODULE_PLTS

Turris community reported to us that they have some issues
with ath9k as it was not loading due to vmalloc allocation failure.
We think that enabling CONFIG_ARM_MODULE_PLTS [1] might help.

This is going to be enabled to all OpenWrt routers, which are
running kernel 4.14.

[1] https://cateee.net/lkddb/web-lkddb/ARM_MODULE_PLTS.html
---
 target/linux/generic/config-4.14 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index 5a4dab9888..f8ab4cc5d7 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -325,7 +325,7 @@ CONFIG_ARM_GIC_MAX_NR=1
 # CONFIG_ARM_KIRKWOOD_CPUFREQ is not set
 # CONFIG_ARM_KPROBES_TEST is not set
 # CONFIG_ARM_MHU is not set
-# CONFIG_ARM_MODULE_PLTS is not set
+CONFIG_ARM_MODULE_PLTS=y
 # CONFIG_ARM_PATCH_PHYS_VIRT is not set
 # CONFIG_ARM_PSCI is not set
 # CONFIG_ARM_PSCI_CHECKER is not set
-- 
2.26.2

